README.md

```diff
@@ -6,7 +6,56 @@
 
 Encoding and decoding support for BSON in Rust
 
+## Useful links
+- [API Documentation](https://docs.rs/bson/)
+- [Serde](https://serde.rs/)
+
+## Installation
+This crate works with Cargo and can be found on
+[crates.io](https://crates.io/crates/bson) with a `Cargo.toml` like:
+
 ```toml
 [dependencies]
 bson = "*"
```

Comment:
Lets keep it as `~0.3.0`? This will auto-upgrade the patch version, but won't upgrade to `0.4.0`, which might contain breaking changes.

Whichever you prefer; I've always found the crate upgrade rules for rust to be a little funky.